packages/bootstrap/scss/_variables.scss

```diff
@@ -3018,6 +3018,7 @@ $gantt-advanced-bg: $success !default;
 $gantt-advanced-bg-lighter: tint($gantt-advanced-bg, 5) !default;
 
 $gantt-action-on-offset-text: #000000 !default;
+$gantt-resize-handler-onoffset-top: 45% !default;
```

Comment:
this should be renamed to `$gantt-resize-handler-on-offset-top` or just `$gantt-resize-handler-offset-top`